back-end/controllers/series_movie_controller.js

```diff
@@ -1,6 +1,7 @@
 const mongoose = require('mongoose');
 const Series = require('../models').Series;
 const Movies = require('../models').Movie;
+var TVMazeService = require('../services/tvmaze.service');
```

Comment:
Should this be a const?

Reply:
Yes, this could be a `const`, but that's not a deal breaker.